bin/download-content-model.js

```diff
@@ -1,42 +1,96 @@
 const fs = require('fs')
 const path = require('path')
+const zlib = require('zlib')
 
-const clone = require('git-clone')
-const ora = require('ora')
+const fetch = require('node-fetch')
+const Listr = require('listr')
+const tar = require('tar')
+const mkdirp = require('mkdirp')
 const rimraf = require('rimraf')
 
 const REPO_ROOT = path.resolve(__dirname, '..')
 const DATA_DIR = path.resolve(REPO_ROOT, 'data')
 
-const permissions = ora('Checking permissions to write files to the disk').start()
-
-fs.access(REPO_ROOT, fs.constants.W_OK, (err) => {
-  if (err) {
-    permissions.fail(`Script is unable to write to project directory. Please check your users file permissions for ${REPO_ROOT}`)
-    process.exit(1)
-  }
-
-  permissions.succeed()
-
-  const cloning = ora('Cloning content model').start()
-
-  rimraf(DATA_DIR, (error) => {
-    if (error) {
-      cloning.fail('An error happend while removing existing content model')
-      process.exit(1)
+const tasks = new Listr([
+  {
+    title: `Checking permissions to write files to the disk`,
+    task: (ctx) => {
+      return new Promise((resolve, reject) => {
+        fs.access(REPO_ROOT, fs.constants.W_OK, (err) => {
+          if (err) {
+            reject()
+            console.error(`Script is unable to write to project directory. Please check your users file permissions for ${REPO_ROOT}`)
+            process.exit(1)
+          }
+          resolve()
+        })
+      })
+    }
+  },
+  {
+    title: `Clean up directory`,
+    task: async (ctx) => {
+      await new Promise((resolve, reject) => {
```

Comment:
This project is not using Babel or anything. Do you think we should use `await` here?

I would like to avoid "await not supported" errors. Currently, there is no `engine` field in the `package.json` defining the version.